### ISSUE TITLE
Japanese island names only appear at one zoom level

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#4578]
I found the actual Japanese island names are only shown at this single zoom level.

Google, and Bing make sure the island name is always shown, if showing any names at all.

https://www.openstreetmap.org/?mlat=30.346254929749488&mlon=130.52473727588293#map=12/30.346254929749488/130.52473727588293

For some reason the township name is shown on OSM with the same frequency that the island name is shown on other mapping services

**Comment by imagico** [comment]
The island label does not show below z12 because it is blocked by a `place=town` label:

https://www.openstreetmap.org/node/582952665

It is not shown at z13+ because the island is too large for that (we stop rendering labels of features once they reach a screen filling size).

That means in this specific case the problem is primarily caused by the mapping error of the above linked `place=town` being misplaced in the middle of the island far away from any settlement.

Independent of that the prioritization of populated places and other labels like for `place=island` will of course need to be considered as part of #3880.

Closing as partly mapping error, partly duplicate of #3880.

**Comment by jidanni** [comment]
Indeed, many such islands consist of exactly one town(ship), whose extent is the whole island. Hence they will always trump the island name, as their centre s are alas, the same.
